### PR TITLE
Align namespaces with current rules

### PR DESCRIPTION
**Pull request by adsteel**
Came across these warnings running Rubocop locally.

### CODE REVIEW

**Comment by pk-nb** [comment]
Curious where you saw these—recent version of rubocop perhaps?

**Review by pk-nb — APPROVED** [review]

**Comment by adsteel** [comment]
Saw these in Napa.

```ruby
be rubocop -v
0.50.0
```

**Comment by bwebster** [comment]
I just upgrade to the most recent rubocop in arugula and see these too.

**Review by bwebster — APPROVED** [review]

File: .rubocop.kapost.yml [hunk 1]
```diff
@@ -33,21 +33,21 @@ Style/AndOr:
...
   EnforcedStyle: context_dependent
-Style/CaseIndentation:
+Layout/CaseIndentation:
   IndentOneStep: true
 Style/Documentation:
   Enabled: true
 Style/EachWithObject:
   Enabled: false
-Style/ExtraSpacing:
+Layout/ExtraSpacing:
```
Comment (bwebster) [inline .rubocop.kapost.yml]: This is duplicated on L98.  Maybe we can combine and fix?

Comment (adsteel) [reply .rubocop.kapost.yml]: Good catch. Combined.

**Review by bwebster — APPROVED** [review]
Thanks for combining.  We should also change `Style/ExtraSpacing` to `Layout/ExtraSpacing`.  Once that is done, I'm on board.

**Comment by adsteel** [comment]
Good catch again @bwebster . Updated.

**Review by bwebster — APPROVED** [review]

**Pull request merged**